src/inputs/TextFieldInput/TextFieldInput.stories.tsx

```diff
@@ -0,0 +1,263 @@
+      />
+    </form>
+  );
+};
+
+export const TextFieldWithErrorsInTheLabel = (): React.ReactElement => {
+  const [value, setValue] = useState<string>('');
+  const [hasError, setHasError] = useState<boolean>(true);
+  const [hiddenLabel, setHiddenLabel] = useState<boolean>(false);
+  const [showErrorsInTheLabel, setShowErrorsInTheLabel] =
+    useState<boolean>(false);
+
+  const error = 'This field has an error';
+
+  return (
+    <form noValidate autoComplete="off" onSubmit={onSubmit}>
+      <Typography>
+        <Switch checked={hasError} onChange={setHasError} />
+        An Error is present
+      </Typography>
+      <Typography>
+        <Switch checked={hiddenLabel} onChange={setHiddenLabel} /> Hide Label
+      </Typography>
+      <Typography>
+        <Switch
+          checked={showErrorsInTheLabel}
+          onChange={setShowErrorsInTheLabel}
+        />
+        Show Errors in The Label
+      </Typography>
+      <TextFieldInput
+        id="standard-TextFieldInput"
+        label="TextFieldInput"
+        name="TextFieldInput"
+        hiddenLabel={hiddenLabel}
+        showErrorsInTheLabel={showErrorsInTheLabel}
+        error={hasError ? error : ''}
+        placeholder={'You can visually hide the label'}
+        value={value}
+        onChange={(e) => setValue(e.target.value)}
+      />
+    </form>
+  );
+};
+
+const StyledTextField = styled(TextFieldInput)`
+  && {
+    .MuiFilledInput-root {
+      background-color: lightgreen;
+      width: 200px;
+      transition: width 1s ease-out;
+    }
+
+    .MuiFilledInput-root.Mui-focused {
+      width: 400px;
+    }
+
+    .MuiFormLabel-root.Mui-focused {
+      color: ${({ error, theme }) =>
+        error ? theme.colors.error : 'darkgreen'};
+    }
+
+    .MuiInputLabel-filled {
+      color: ${({ theme, error }) => (error ? theme.colors.error : 'purple')};
+    }
+
+    .MuiFilledInput-underline:after {
+      border-bottom: 2px solid
+        ${({ theme, error }) => (error ? theme.colors.error : 'orange')};
+    }
+  }
+`;
+
+export const CustomCSSTextField = (): React.ReactElement => {
+  const [value, setValue] = useState<string>('Focus me! :D');
+
+  // const StyledTextField = styled(TextFieldInput)`
+  //   && {
+  //     .MuiFilledInput-root {
+  //       background-color: lightgreen;
+  //       width: 200px;
+  //       transition: width 1s ease-out;
+  //     }
+
+  //     .MuiFilledInput-root.Mui-focused {
+  //       width: 400px;
+  //     }
+
+  //     .MuiFormLabel-root.Mui-focused {
+  //       color: ${({ error, theme }) =>
+  //         error ? theme.colors.error : 'darkgreen'};
+  //     }
+
+  //     .MuiInputLabel-filled {
+  //       color: ${({ theme, error }) => (error ? theme.colors.error : 'purple')};
+  //     }
+
+  //     .MuiFilledInput-underline:after {
+  //       border-bottom: 2px solid
+  //         ${({ theme, error }) => (error ? theme.colors.error : 'orange')};
+  //     }
+  //   }
+  // `;
```

Comment:
Should this be removed?

Reply:
I added that comment to be able to see the styles in the storybook:

![Captura de pantalla 2021-11-22 a las 16 36 41](https://user-images.githubusercontent.com/26763673/142891131-cf7c1b11-cfdc-4551-8193-a42c54462d3c.png)

But now I think that its better add the code inside a `<pre>` tag:
![Captura de pantalla 2021-11-22 a las 16 42 00](https://user-images.githubusercontent.com/26763673/142891213-f6345f31-336c-4f3d-a4c5-b3a17d010dbc.png)